lib/services/northBound/contextServer.js

```diff
@@ -1119,6 +1412,21 @@ function queryErrorHandlingNgsi2(error, req, res, next) {
     });
 }
 
+function queryErrorHandlingNgsiLD(error, req, res, next) {
```

Comment:
Missing JavaDoc?

Reply:
Fixed c70d9ed

Reply:
Maybe github diff is fooling me, but it seems JavaDoc was not added here, despite the "Fixed ..." mark.

![imagen](https://user-images.githubusercontent.com/1534240/75545636-263d5700-5a27-11ea-9e33-01a74c6b7cf4.png)

Can you check, please?

Reply:
Fixed  a5c650a